salt/apps/ocw/symlinks_cms.sls

```diff
@@ -0,0 +1,82 @@
+
+symlink_prod_new:
+  file.symlink:
+    - name: /prod_new
+    - target: /mnt/ocwfileshare
+    - force: True
+    - user: root
+    - group: root
+
+symlink_prod:
+  file.symlink:
+    - name: /prod
+    - target: /prod_new
+    - force: True
+    - user: root
+    - group: root
+
+symlink_files_ocwfileshare:
+  file.symlink:
+    - name: /files/OCWFileShare
+    - target: /mnt/ocwfileshare
+    - force: True
+    - makedirs: True
+    - owner: root
+    - group: root
+
+symlink_ocwuser_ocwengines:
+  file.symlink:
+    - name: /home/ocwuser/OCWEngines
+    - target: /mnt/ocwfileshare/OCWEngines
```

Comment:
Is this actually necessary? Are there any cases where that code is being used from the ocwuser home directory? If not I'd prefer to avoid muddying the waters further...